go/stellar/stellarsvc/frontend_test.go

```diff
@@ -1430,7 +1430,7 @@ func TestBuildPaymentLocal(t *testing.T) {
 	t.Logf(spew.Sdump(bres))
 	require.Equal(t, false, bres.ReadyToReview)
 	require.Equal(t, "", bres.ToErrMsg)
-	require.Equal(t, "You have *0 XLM* available to send.", bres.AmountErrMsg)
+	require.Equal(t, "You only have *0 XLM* available to send.", bres.AmountErrMsg)
```

Comment:
I think for the zero case "only" sounds weird.  @malgorithms ?

Reply:
yeah, agreed.

Reply:
ok i'll make it 3 cases then